main.dart

```diff
@@ -1,3 +1,58 @@
 void main() {
```

Comment:
please, make user friendly output

9
21
Phone created is 3


is not good enough